snarkos/node.rs

```diff
@@ -41,21 +38,25 @@ impl<N: Network, E: Environment> Node<N, E> {
         // Initialize the ledger.
         let ledger = Ledger::<N>::load(account.private_key()).await?;
 
+        // If the node is not in development mode, perform fast sync with the network.
+        if cli.dev.is_none() {
```

Comment:
why is fast sync not done in development mode?

Reply:
dev mode is usually dedicated to running a few nodes locally, so syncing with the actual leader might be undesirable

Reply:
gotcha, thanks